packages/monorepo-scripts/src/publish.ts

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env node
+
+import * as fs from 'fs';
+import lernaGetPackages = require('lerna-get-packages');
+import * as _ from 'lodash';
+import * as moment from 'moment';
+import * as path from 'path';
+import { exec as execAsync, spawn } from 'promisify-child-process';
+import semverDiff = require('semver-diff');
+import semverSort = require('semver-sort');
+
+import { constants } from './constants';
+import { Changelog, Changes, SemVerIndex, UpdatedPackage } from './types';
+import { utils } from './utils';
+
+const IS_DRY_RUN = false;
+const TODAYS_TIMESTAMP = moment().unix();
+const LERNA_EXECUTABLE = './node_modules/lerna/bin/lerna.js';
+const semverNameToIndex: { [semver: string]: number } = {
+    patch: SemVerIndex.Patch,
+    minor: SemVerIndex.Minor,
+    major: SemVerIndex.Major,
+};
+
+(async () => {
+    const updatedPublicPackages = await getPublicLernaUpdatedPackagesAsync();
+    const updatedPackageNames = _.map(updatedPublicPackages, pkg => pkg.name);
+
+    const allLernaPackages = lernaGetPackages(constants.monorepoRootPath);
+    const relevantLernaPackages = _.filter(allLernaPackages, pkg => {
```

Comment:
If relevant package is a package that is updated why don't we call it updatedLernaPackages and updatedPackageNames. I don't fully understand that step.